manifest.json

```diff
@@ -20,12 +20,14 @@
 	"content_scripts": [
 		{
 			"matches": ["http://*/*", "https://*/*"],
+			"match_about_blank": true,
 			"run_at": "document_start",
 			"all_frames": true,
 			"js": ["apply.js"]
 		},
 		{
 			"matches": ["http://userstyles.org/*", "https://userstyles.org/*"],
+			"match_about_blank": true,

```

Comment:
This block is to handle installs from userstyles.org. It shouldn't have `match_about_blank`.